grails-app/views/picklist/_page.gsp

```diff
@@ -98,10 +98,9 @@
                         </g:if>
                     </td>
                     <td class="center middle">
-                        <g:set var="binLocation" value="${requisitionItem?.product?.getInventoryLevel(location?.id)?.binLocation}"/>
-                        <g:each in="${binLocation?.split(';')}" var="binLocationPart">
-                            ${binLocationPart}<br/>
-                        </g:each>
+                        <g:if test="${picklistItems}">
+                            ${picklistItems[j]?.binLocation?.name}
+                        </g:if>
```

Comment:
for me the issue was that 
```
requisitionItem?.product?.getInventoryLevel(location?.id)?.binLocation
```
returned a `null` value so I checked how is this value accessed in picklist print page and followed that example
```
 <td class="center middle">
    <g:if test="${picklistItems}">
        <div class="binLocation">
            ${picklistItems[j]?.binLocation?.name}
        </div>
    </g:if>
</td>
```
https://github.com/openboxes/openboxes/blob/develop/grails-app/views/picklist/_printPage.gsp#L124

Reply:
The following has nothing to do with the picklist item. It is a configuration on the InventoryLevel in case there is a static bin location where stock should be stored. 
```
requisitionItem?.product?.getInventoryLevel(location?.id)?.binLocation
```

Can you upload a screenshot / PDF of the picklist report as it currently stands?

Reply:
> I believe the bin location is not supposed to be null and should match the table from the print page

My point was that this
```
requisitionItem?.product?.getInventoryLevel(location?.id)?.binLocation
```
and this
```
${picklistItems[j]?.binLocation?.name}
```
are two completely different things. The former is showing the default bin location for a particular product at a particular facility. The other is telling the user where to find the item that needs to be picked. 

So there's something odd going on here. Under the picklist directory there's _page.gsp, _print.gsp, _printPage.gsp, print.gsp. 

We'll need to review this at a later time but for now I suppose it's safe to replace the default bin location with
```
${picklistItems[j]?.binLocation?.name}
```

Reply:
My guess is that the Print PDF feature that is associated with this GSP hasn't been used often. Up until 2018, picklist item did not include a bin location so feature was just showing where we expected the stock to be. When we added bin location this file was not updated because it wasn't used often and we probably forgot about it.